tests/unit/Change.spec.ts

```diff
@@ -158,7 +162,7 @@ describe('Change component', () => {
       }))
 
     // GET business share classes
-    get.withArgs('businesses/BC1234567/share-classes')
+    get.withArgs('businesses/FM1234567/share-classes')
```

Comment:
This isn't used by Firm Registration Corrections, so you can remove it. Same with name translations, resolutions, ... (check what the change filing snapshot fetches).

Reply:
Okay should I remove the test checks from the xit tests that are marked to be completed in the future?